src/librustc/middle/dependency_format.rs

```diff
@@ -210,6 +210,8 @@ fn calculate_type(tcx: TyCtxt<'_>, ty: config::CrateType) -> DependencyList {
     // quite yet, so do so here.
     activate_injected_dep(*sess.injected_panic_runtime.get(), &mut ret,
                           &|cnum| tcx.is_panic_runtime(cnum));
+    activate_injected_dep(*sess.injected_sanitizer_runtime.get(), &mut ret,
+                          &|cnum| tcx.is_sanitizer_runtime(cnum));
```

Comment:
I am not sure if this is necessary. It would only really affect linux targets which I don't have access to for testing. My thinking was changing the sanitizer crate depkind to `Implicit` requires adding `activate_injected_dep` in suitable places.